dora/core/common/src/test/java/alluxio/UfsUrlTest.java

```diff
@@ -118,6 +138,127 @@ public void hashCodeTest() {
     assertEquals(ufsUrl2.hashCode(), ufsUrl3.hashCode());
   }
 
+  @Test
+  public void pathWithWhiteSpacesNoException() throws IllegalArgumentException {
+    String[] paths = new String[]{
+        "file:/// ",
+        "file:///  ",
+        "file:/// path",
```

Comment:
what about `  file:  / / /path` etc.?

Reply:
an interesting question😅. I think it should be illegal.

Reply:
added in UT.